x-pack/plugin/analytics/src/test/java/org/elasticsearch/xpack/analytics/mapper/HistogramFieldMapperTests.java

```diff
@@ -49,6 +49,11 @@ protected void registerParameters(ParameterChecker checker) throws IOException {
             m -> assertTrue(((HistogramFieldMapper)m).ignoreMalformed()));
     }
 
+    @Override
+    protected boolean supportsScripts() {
```

Comment:
should we unify naming between this and `allowsIndexTimeScript` ?

Reply:
So this is not actually about index time scripts, it's about access to field data via a lookup.  Maybe it should be `supportsLookup` instead?